src/disco/gui/fd_gui.c

```diff
@@ -1442,6 +1443,20 @@ fd_gui_handle_start_progress( fd_gui_t *    gui,
   fd_http_server_ws_broadcast( gui->http );
 }
 
+static void
+fd_gui_handle_genesis_hash( fd_gui_t * gui,
```

Comment:
```suggestion
fd_gui_handle_genesis_hash( fd_gui_t *    gui,
```

More annoying style stuff, not important